acme/acme/challenges_test.py

```diff
@@ -269,7 +269,7 @@ def setUp(self):
             'validation': self.validation.to_json(),
         }
 
-        # pylint: disable=invalid-name
+
```

Comment:
Unnecessary newline. It looks like there are a few of these wherever `invalid-name` was the only option being temporarily disabled in `pylint`.